lib/app/features/auth/providers/onboarding_complete_notifier.r.dart

```diff
@@ -102,7 +102,8 @@ class OnboardingCompleteNotifier extends _$OnboardingCompleteNotifier {
         await _sendFollowListToFollowees(
```

Comment:
Can't we reuse the existing `userEventsMetadataBuilderProvider`?
So `_sendFollowListToFollowees` could look like this:
```
final userEventsMetadataBuilder = await ref.read(userEventsMetadataBuilderProvider.future);
final ionNotifier = ref.read(ionConnectNotifierProvider.notifier);
await Future.wait(
  followList.list.map(
    (followee) => ionNotifier.sendEntityData(
      followList,
      actionSource: ActionSourceUser(followee.pubkey),
      metadataBuilders: [userEventsMetadataBuilder],
      cache: false,
    ),
  ),
);
```